pkg/storage/export/export/export.go

```diff
@@ -617,13 +618,17 @@ func (ctrl *VMExportController) handleVMExportSecret(vmExport *exportv1.VirtualM
 	}
 
 	// The secret name is constructed so it can be specific to the current vmExport object
-	tokenSecretName := getDefaultTokenSecretName(vmExport)
+	updateDefaultTokenSecretName(vmExport)
 
-	token := rand.String(secretTokenLenght)
+	// Securely randomize a 20-bytes token
+	token := make([]byte, secretTokenLenght)
+	if _, err := cryptorand.Read(token); err != nil {
+		return err
+	}
```

Comment:
We still want this to be string.  The value has to be included in an http header.  So maybe do something like this:  https://gist.github.com/dopey/c69559607800d2f2f90b1b1ed4e550fb#file-main-go-L47-L59

Basically whatever the k8s library function was doing but with a better random source